Documentation/Registry.md

```diff
@@ -1031,6 +1032,14 @@ Content-Language: en
 }
 ```
 
+##### 4.6.2.1. Reserved metadata keys
+
+These metadata keys have special meanings to the server and should be included if and only if their value satisifies their intended use:
+
+| Key               | Description                                          | Server Use                             |
+| ----------------- | ---------------------------------------------------- | -------------------------------------- |
+| `repositoryURL`   | HTTPS URL of the package's source code repository.   | The mapping between package identifer and repository URL gets recorded for the [lookup package identifiers by URL]((#endpoint-5) endpoint. | 
```

Comment:
should this allow more than one URL, for example `apple.swift-nio` should be mappable to both `https://github.com/apple/swift-nio.git` and `git@github.com:apple/swift-nio.git` and in the enterprise case there may be even more URLs

Reply:
amended https://github.com/apple/swift-package-manager/pull/4221/commits/017ed80ca3ae9af4e70a1c0ff0780ecaccb3870d